vcpkg-configuration.json

```
@@ -7,12 +7,12 @@
     }
   ],
   "requires": {
-    "arm:tools/kitware/cmake": "^3.25.2",
-    "arm:tools/ninja-build/ninja": "^1.10.2",
-    "arm:compilers/arm/armclang":"^6.20.0",
-    "arm:compilers/arm/arm-none-eabi-gcc": "^13.2.1",
-    "arm:compilers/arm/llvm-embedded": "^17.0.1-0",    
-    "arm:tools/open-cmsis-pack/cmsis-toolbox": "^2.0.0-0",
-    "arm:models/arm/avh-fvp": "^11.22.39"
+    "arm:tools/kitware/cmake": "^3.28.4",
+    "arm:tools/ninja-build/ninja": "^1.12.0",
+    "arm:compilers/arm/armclang":"^6.22.0",
+    "arm:compilers/arm/arm-none-eabi-gcc": "^13.3.1",
+    "arm:compilers/arm/llvm-embedded": "18.1.3",    
```

```suggestion
    "arm:compilers/arm/llvm-embedded": "^18.1.3",    
```
The `^` does not allow update to another major version but only picks up minor or patch updates, automatically. If we want to use any later version, we'd need to specify `>=18.1.3` or just `>=17.0.1-0`.